ci/docker/runtime_functions.sh

```diff
@@ -1212,6 +1230,8 @@ unittest_ubuntu_cpugpu_perl() {
 
 unittest_cpp() {
     set -ex
+    export PYTHONPATH=./python/
+    python3 -c "import mxnet as mx; mx.test_utils.download_model(\"imagenet1k-resnet-18\"); mx.test_utils.download_model(\"imagenet1k-resnet-152\"); mx.test_utils.download_model(\"imagenet1k-resnet-50\");"
```

Comment:
I don't like this inlined python very much, it's also duplicated code below, it could be factored out into a utility script or a function.

Reply:
I think writing a function for this is an overkill.